policyengine_us/variables/gov/states/al/tax/income/exemptions/retirement/al_retirement_exemption_person.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class al_retirement_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Alabama retirement exemption for each person"
+    unit = USD
+    # The Code of Alabama Section 40-18-19 (a)(13).
+    documentation = "https://casetext.com/statute/code-of-alabama/title-40-revenue-and-taxation/chapter-18-income-taxes/article-1-general-provisions/section-40-18-19-exemptions-generally"
+    definition_period = YEAR
+    defined_for = "al_retirement_exemption_eligible_person"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.al.tax.income.exemptions.retirement
+        retirement_income = add(
+            person,
+            period,
+            [
+                "taxable_ira_distributions",
```

Comment:
taxable_retirement_distributions includes the 4 categories
```suggestion
                "taxable_retirement_distributions", "taxable_pension_income"]
```